curvine-common/src/state/worker_address.rs

```diff
@@ -49,3 +50,27 @@ impl Display for WorkerAddress {
         )
     }
 }
+
+impl From<WorkerAddress> for WorkerAddressProto {
```

Comment:
Use the functions of ProtoUtils to handle address conversion.
Strictly speaking, Proto data structures should only appear in one place and should not be used elsewhere.

Reply:
done